en/docs/streaming/migrating-from-stream-processor.md

```diff
@@ -0,0 +1,45 @@
+# Migrating from WSO2 Stream Processor
+
+The Streaming Integrator performs all functions that are also performed by [WSO2 Stream Processor](https://docs.wso2.com/display/SP440/Stream+Processor+Documentation). It also has additional features to trigger integration flows in order to take action in response to results derived after analyzing data.
+
+If you are currently using WSO2 Stream Processor to carry out any streaming integration/stream processing activities and want to carry them out in the Streaming Integrator, you can migrate your setup as follows:
+
+## Preparing to upgrade
```

Comment:
Is it better to put the sub-heading as Step 1,2,3 etc. Example, 
- Before you begin
- Step 1 - Migrate the databases